addons/website/static/src/builder/plugins/floating_blocks/floating_blocks_block_mobile_option.js

```diff
@@ -1,13 +1,12 @@
 import { BaseOptionComponent, useDomState } from "@html_builder/core/utils";
-import { isMobileView } from "@html_builder/utils/utils";
 
 export class FloatingBlocksBlockMobileOption extends BaseOptionComponent {
     static template = "website.FloatingBlocksBlockMobileOption";
     static props = {};
     setup() {
         super.setup();
         this.state = useDomState((editingElement) => ({
-            isMobileView: isMobileView(editingElement),
+            isMobileView: this.config.isMobileView(editingElement),
```

Comment:
`this.config` is not defined here, as we are in a component, you need to get the editor from the env first :wink: 
=> `this.env.editor.config`